.github/workflows/e2e_tests.yml

```diff
@@ -0,0 +1,281 @@
+    runs-on: ${{ matrix.os }}
+    needs: [create-test-images]
+
+    env:
+      HUB: hellozee-docker-istio-testing.bintray.io # change
+
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+
+      - name: Set up Cloud SDK
+        uses: azure/login@v1
+        with:
+          creds: ${{ secrets.AZURE_CREDENTIALS }} 
+
+      - name: set up go 1.15
+        uses: actions/setup-go@v1
+        with:
+          go-version: 1.15
+
+      - name: aks-cluster-create
+        run: bash ./tetrateci/create_aks_cluster.sh
+
+      - name: integration-tests
+        run: |
+          export TAG=$(git describe --tags)
+          go test -tags=integ ./tests/integration/operator/...  -istio.test.skipVM true -p 1  -test.v
+          go test -tags=integ ./tests/integration/helm/... -istio.test.skipVM true -p 1 -test.v
+          go test -tags=integ -timeout 30m -run='TestEmptyCluster|TestFileOnly|TestDirectoryWithoutRecursion|TestDirectoryWithRecursion|TestInvalidFileError|TestJsonInputFile|TestJsonOutput|TestKubeOnly|TestFileAndKubeCombined|TestAllNamespaces|TestTimeout|TestErrorLine|TestWait|TestVersion|TestDescribe|TestAddToAndRemoveFromMesh|TestProxyConfig|TestProxyStatus|TestAuthZCheck|TestLocality|TestMain|TestMirroring|TestMirroringExternalService|TestTproxy|TestValidation|TestEnsureNoMissingCRDs|TestWebhook' ./tests/integration/pilot/ -istio.test.skipVM true -p 1 -test.v 
+          go test -tags=integ ./tests/integration/pilot/analysis/... -istio.test.skipVM true -p 1 -test.v
+          go test -tags=integ ./tests/integration/pilot/revisions/... -istio.test.skipVM true -p 1 -test.v
+          go test -tags=integ -timeout 30m -run='TestStatsFilter|TestStatsTCPFilter|TestSetup|TestIstioctlMetrics|TestTcpMetric|TestStatsFilter|TestWASMTcpMetric|TestWasmStatsFilter|TestMain|TestCustomizeMetrics'  ./tests/integration/telemetry/stats/... -istio.test.skipVM true -p 1 -test.v
+          go test -tags=integ  ./tests/integration/security/ca_custom_root/... -istio.test.skipVM true -p 1 -test.v
+          go test -tags=integ  ./tests/integration/security/file_mounted_certs/... -istio.test.skipVM true -p 1 -test.v
+          go test -tags=integ  ./tests/integration/security/filebased_tls_origination/... -istio.test.skipVM true -p 1 -test.v
+          go test -tags=integ  ./tests/integration/security/mtls_first_party_jwt/... -istio.test.skipVM true -p 1 -test.v
+          go test -tags=integ  ./tests/integration/security/mtlsk8sca/... -istio.test.skipVM true -p 1 -test.v
+          go test -tags=integ  ./tests/integration/security/sds_egress/... -istio.test.skipVM true -p 1 -test.v
+          go test -tags=integ  ./tests/integration/security/sds_tls_origination/... -istio.test.skipVM true -p 1 -test.v
+          go test -tags=integ  ./tests/integration/security/webhook/... -istio.test.skipVM true -p 1 -test.v
+
+      - name: aks-cluster-delete
+        run: bash ./tetrateci/cleanup_aks_cluster.sh
+    
+  create-new-tag:
```

Comment:
We need the release as a separate workflow so that it can also be triggered manually. In this create-new-tag job, we can trigger the release workflow with the SHA. Can you check if we can use workflow_dispatch (if not repository_dispatch) for the release workflow.

The need for making a manual release is very common - emergency security patch fix, integration tests fails for false reasons, etc. In those cases, a manual way to trigger the release job with the tag/commit-sha is needed